bindings/go/oci/internal/log/logging_test.go

```diff
@@ -0,0 +1,80 @@
+package log
+
+import (
+	"log/slog"
+	"testing"
+	"time"
+
+	ociImageSpecV1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/stretchr/testify/assert"
+
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+func TestOperation(t *testing.T) {
+	ctx := t.Context()
+
+	// Test successful operation
+	done := Operation(ctx, "test-operation", slog.String("test", "value"))
+	time.Sleep(10 * time.Millisecond) // Simulate some work
+	done(nil)                         // No error
+
+	// Test failed operation
+	done = Operation(ctx, "test-operation-failed", slog.String("test", "value"))
+	time.Sleep(10 * time.Millisecond) // Simulate some work
+	done(assert.AnError)              // With error
+}
```

Comment:
What should this test test? It is only calling functions without testing for any result

Reply:
agreed, will add a test for the content of the log message in stdout